docs/source/contributors/structure.rst

```diff
@@ -31,26 +31,23 @@ The specific tests run are defined in a series of Bash scripts,
 which are stored in this folder.
 
 ###################
-Docker (``Docker``)
+Container (``Container``)
 ###################
```

Comment:
It's very minor but the title over- and underline here is too short which causes Sphinx to throw a warning when building the docs